docs/pages/api-reference/liveblocks-react-ui.mdx

```diff
@@ -351,6 +358,9 @@ Learn more about mutation hooks under
   <PropertiesListItem name="defaultValue" type="CommentBody">
     The composer’s initial value.
   </PropertiesListItem>
+  <PropertiesListItem name="defaultAttachments" type="CommentAttachment[]">
+    The composer's initial attachments.
+  </PropertiesListItem>
```

Comment:
There were a few unlisted props that should be documented so I added them while I was there.